### ISSUE TITLE
Increment chart version for more clarity

### ISSUE THREAD

**Issue by wigeric** [cockroachdb/helm-charts#221]
For a while now, cockroachdb version is bumped but chart version stays at 7.0.0.

It becomes quite a nightmare to manage.

Could you increase chart version at least for each app version update ?

**Comment by davidwding** [comment]
Hi, thanks for opening an issue. Are you asking specifically about `version` and `appVersion` in cockroachdb/Chart.yaml? The two fields are independent and can be changed separately, in accordance in the [Helm docs](https://helm.sh/docs/topics/charts/#the-appversion-field). As such, we don't currently see a strong reason to bump `version` whenever `appVersion` is bumped.

To help us understand your use case better, can you elaborate on how the current behavior is making your workflow more difficult?

**Comment by wigeric** [comment]
Hi, I do get that chart version and appVersion are independant, however in all the other charts that I use, every change in the chart causes a chart version bump, including appVersion change but other changes too.

In my case : I use fluxCD to manage my middlewares. I got 1 file that manages cockroachDB for 9 clusters. 
FluxCD compares won't download 
For every other chart I use, I only need to change the chart version on that 1 file to upgrade my middleware everywhere. Without the chart version bump, I need to connect to every cluster, delete the known helm chart to force fluxCD to download it.
And it is not a problem from fluxCD as I would not want all my cockroachDB to upgrade silently because the image has been changed in the same chart version.

Chart version bump also permits to rollback app version without having to manage the image tag in values file.

There has been at least 10 PR since last chart version change, that doesn't seems optimal to me in terms of traceability.
